src/malloc.c

```diff
@@ -1700,7 +1694,9 @@ _pma_get_cached_dpage(void) {
 
   // TODO: macros for dealing with cache?
   // Pop page off queue
+  uint16_t head   = _pma_state->metadata.dpage_cache->head;
```

Comment:
Do you recall the reason for moving this declaration down? It's not used between the location in the original code and the location in this PR.